keras/src/layers/core/lambda_layer.py

```diff
@@ -215,7 +215,7 @@ def from_config(cls, config, custom_objects=None, safe_mode=None):
                 )
                 config["output_shape"] = fn
             else:
-                config["output_shape"] = (
+                config["output_shape"] = tuple(
```

Comment:
There may be cases where the shape is not a tuple but a list of tuples (etc.). We need to check whether the tuple only contains int/None first.